srv/relayer.go

```diff
@@ -120,15 +122,39 @@ func (r *Relayer) SendUserOperation() modules.BatchHandlerFunc {
 			attribute.Int64("wait_timeout", int64(r.waitTimeout)),
 		)
 
+		var computeOPHashes = func(err error, hash common.Hash) {
+
+			for _, op := range ctx.Batch {
+
+				currentOpHash, unsolvedOpHash := utils.GetUserOpHash(op, r.ep, r.chainID)
+
+				r.m.Compute(unsolvedOpHash, func(oldValue OpHashes, loaded bool) (newValue OpHashes, delete bool) {
+
+					var mergedErr error
+					if err != nil {
+						mergedErr = multierror.Append(oldValue.Error, err)
+					}
+
+					return OpHashes{
+						Error:  mergedErr,
+						Solved: currentOpHash,
+						Trx:    hash,
+					}, false
+				})
+			}
+		}
```

Comment:
I like agree with coderabbit's comment :) I found it hard to follow which variable is named vs captured and  will improve the readability by using named parameters (ctx.Batch vs ops) and avoid 2 level closures.